includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -1239,12 +1243,14 @@ public function is_renew_action() {
 	 */
 	public function remove_edit_steps_for_renewal( $steps ) {
 		unset( $steps['submit'], $steps['preview'] );
-		$steps['renew-listing'] = [
-			'name'     => 'Renew Listing',
-			'view'     => [ $this, 'preview' ],
-			'handler'  => [ $this, 'renew_preview_handler' ],
-			'priority' => 29, // This action should be just before 'done' whose priority is 30.
-		];
+		if ( ! class_exists( 'WC_Paid_Listings' ) && ! class_exists( 'WP_Job_Manager_Simple_Paid_Listings' ) ) {
```

Comment:
Here we add preview step only if there's no paid listing plugin activated. If there's a paid listing, renewing the listing will happen in the plugin (WC paid listings) or in the `done_before` step (Simple paid listing).